### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by andreschaffer**
Make the readme easier and more straightforward to read.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,65 +1,73 @@
...
 
-Within Gogs, we create two git repositories: the microservice_code_generator, which is self-explanatory, and the jenkins_seed, where the Jenkins Job-DSL definitions will live. The Jenkins Job-DSL plugin allows for having pipelines as code, in case you were not familiar with it.  
+We use Docker to run Jenkins and Gogs. Jenkins itself has access to Docker in a Docker-outside-of-Docker approach, i.e. we map the host's Docker socket, so that containers started by Jenkins will be siblings.
 
-Now, (our second inception...) using a seed job in Jenkins with the jenkins_seed repository as remote url (origin), we create the other jobs involved in this solution. Nice! The seed job's duty does not end here though, as it's responsible for keeping Jenkins pipelines updated as things evolve.  
+Being Gogs our Git Server, we push two repositories into it:
```
Comment (tommysdk) [inline README.md]: Suggestion:
"We have prepared to two repositories into our Git server (Gogs):"

File: README.md [hunk 2]
```diff
@@ -1,65 +1,73 @@
...
 
-Within Gogs, we create two git repositories: the microservice_code_generator, which is self-explanatory, and the jenkins_seed, where the Jenkins Job-DSL definitions will live. The Jenkins Job-DSL plugin allows for having pipelines as code, in case you were not familiar with it.  
+We use Docker to run Jenkins and Gogs. Jenkins itself has access to Docker in a Docker-outside-of-Docker approach, i.e. we map the host's Docker socket, so that containers started by Jenkins will be siblings.
 
-Now, (our second inception...) using a seed job in Jenkins with the jenkins_seed repository as remote url (origin), we create the other jobs involved in this solution. Nice! The seed job's duty does not end here though, as it's responsible for keeping Jenkins pipelines updated as things evolve.  
+Being Gogs our Git Server, we push two repositories into it:  
+- microservice_code_generator, which is responsible for the generation of the microservices' skeleton.  
+We are creating simple Python Flask microservices in this project but it should be fairly simple for you to adapt it to your needs.
```
Comment (tommysdk) [inline README.md]: Suggestion:
"In our case, we are just creating a simple Python Flask application as our microservice but you can simply adjust it to your needs (being the code skeleton relevant for your context and with your programming language of choice)."

File: README.md [hunk 3]
```diff
@@ -1,65 +1,73 @@
...
+- jenkins_seed, where we have our pipelines as code via Jenkins Job-DSL definitions.
 
-From that point, we already have a create_microservice pipeline that fulfills the promise and allows us to create microservices with.one.click! Really cool. How it does it, well, you can bet there's more inception coming... and you'd win. :)  
+One of those pipelines that we have is the create_microservice pipeline that fulfills the promise and creates microservices with.one.click. Awesome. What it does:
```
Comment (tommysdk) [inline README.md]: Suggestion:
"One of the pipelines is the create_microservice pipeline that fulfills the promise of creating microservices with one click. What it does:"

File: README.md [hunk 4]
```diff
@@ -1,65 +1,73 @@
...
+Good examples of extending the create_microservice pipeline are adding steps to create metrics dashboards and monitoring alerts for your new microservice.  
+An obvious example of adapting the microservice's pipeline is changing how / where it deploys the service.
 
-Phew. Now we are done :)  
-Let's see how to put it in action next!
+Let's get hands on next!
 
-## Prerequisites
-- Make sure you have [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) (to be able to clone this repo), [Docker](https://docs.docker.com/engine/installation/) & [Docker Compose](https://docs.docker.com/compose/install/) installed
+## Pre-requisites
```
Comment (tommysdk) [inline README.md]: Let's use "Prerequisites" instead

File: README.md [hunk 5]
```diff
@@ -1,65 +1,73 @@
...
 
 ## Going forward
-Now that you have a solution for easily creating microservices, one thing will happen: you WILL create lots of microservices. And in order to be able to maintain them, you'll need the proper mindset. A few tips:  
-- Grow your system with convention in mind (i.e. for builds, deployments, monitoring, etc)  
-- Adopt a test strategy that scales (there is one here that you can follow: https://github.com/andreschaffer/microservices-testing-examples)  
-- Put effort into identifying the right microservices boundaries (strategic [Domain-Driven Design](https://en.wikipedia.org/wiki/Domain-driven_design) can take you a long way)  
-- Design for failure (because shit happens)  
+Now that you have a solution for easily creating microservices, one thing will happen: you WILL create lots of microservices. And in order to be able to maintain them, you'll need the proper mindset. A few tips:
+- Grow your system with convention in mind (e.g. for builds, deployments, monitoring, etc)
+- Adopt a test strategy that scales (there is one here that you can follow: https://github.com/andreschaffer/microservices-testing-examples)
```
Comment (tommysdk) [inline README.md]: Make an explicit link to the microservices-testing-examples repository!

File: README.md [hunk 6]
```diff
@@ -1,65 +1,73 @@
...
+- Put effort into identifying the right microservices boundaries (strategic [Domain-Driven Design](https://en.wikipedia.org/wiki/Domain-driven_design) can take you a long way)
+- Design for failure (because shit happens)
```
Comment (tommysdk) [inline README.md]: Change to "because shit will happen"?

**Review by tommysdk — CHANGES_REQUESTED** [review]
Added some suggestions for rephrasing some sentences. Good initiative to make the documentation more straight forward!

File: README.md [hunk 7]
```diff
@@ -14,12 +14,12 @@ The central pieces of this solution are [Docker](https://www.docker.com/), [Jenk
 
 We use Docker to run Jenkins and Gogs. Jenkins itself has access to Docker in a Docker-outside-of-Docker approach, i.e. we map the host's Docker socket, so that containers started by Jenkins will be siblings.
 
-Being Gogs our Git Server, we push two repositories into it:  
+We have prepared to two repositories into Gogs (our Git server):
```
Comment (tommysdk) [inline README.md]: "to two" - remove "to"

**Review by tommysdk — APPROVED** [review]

File: README.md (hunk 7 above)
Comment (andreschaffer) [reply README.md]: That was your suggestion! Hehehe!

**Pull request merged**